README.md

```diff
@@ -34,6 +38,10 @@ can change this with the --config option.
 
 Once you're in the REPL, try typing `help` to see what you can do.
 
+## Docker
+
+`docker run --rm -it -v $HOME/.kube/:/root/.kube/ click`
```

Comment:
Could you add a note that if any of your certificates are stored outside the `.kube` dir you'll need to bind that in as well?

Reply:
There's also the issue of kubectl configurations that use other commands to gather auth info (e.g. gcloud for GKE clusters).